pulsar/managers/base/__init__.py

```diff
@@ -384,6 +385,15 @@ def get_mapped_file(directory, remote_path, allow_nested_files=False, local_path
         if mkdir and not local_path_module.exists(local_directory):
```

Comment:
Is there any difference in these methods? If not - don't you want to delete this one and import that other one?

Reply:
I assumed you had some good reason for it being copy/pasted. 😂